### DISCUSSION TITLE
Plugin not visible on the available plugins list

### DISCUSSION THREAD

**Discussion by Massad** [fastlane/fastlane#20094] [Q&A]
Hello,

`fastlane-plugin-testappio` plugin is not visible in [available plugins ](https://docs.fastlane.tools/plugins/available-plugins/)

Gem:
https://rubygems.org/gems/fastlane-plugin-testappio

Repo:
https://github.com/testappio/fastlane-plugin-testappio

I've looked in many places but couldn't find a reason not to, can someone suggest if there's a missing step?

**Comment by Kukurijek** [comment]
Hi @Massad!
The problem is, that the deploy job in the last pull request has failed. ->
https://github.com/fastlane/docs/pull/1146

Just wait for the next healthy docs release 😄

**Reply by Massad** [reply]
Thanks @Kukurijek for the info! I was looking everywhere for a solution.

I will wait for the update then 😄

**Comment by Massad** [comment]
Any suggestions why this plugin is not in list yet? It's been since 20 Mar 2022 :)
